packages/editor/src/client/atoms/currencyInput.tsx

```diff
@@ -1,17 +1,16 @@
 import React, {useState, useEffect} from 'react'
-import {Input, InputGroup, Message} from 'rsuite'
-import {useTranslation} from 'react-i18next'
+import {Form, InputGroup} from 'rsuite'
 
 export interface CurrencyInputProps {
   currency: string
   centAmount: number
   onChange(centAmount: number): void
   disabled?: boolean
+  name: string
 }
 
 export function CurrencyInput({currency, centAmount, disabled, onChange}: CurrencyInputProps) {
   const [amount, setAmount] = useState<number | string | any>(centAmount / 100)
-  const [message, setMessage] = useState(false)
```

Comment:
What's the idea about this change here? :)

Reply:
The error message now comes from the validation, this one is not needed anymore